resources/public/api/conf/1.0/endpoints/delete_benefit.yaml

```diff
@@ -0,0 +1,93 @@
+summary: Delete State Benefit
+description: |
+  This endpoint allows a developer to delete customer submitted state benefits.
```

Comment:
The description in the [spec](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=207457230) is slightly different:
`
This endpoint allows a developer to delete customer submitted state benefits. If the customer attempts to delete a pre-populated state benefit then the operation will fail with a forbidden response.

A National Insurance number, tax year and benefit ID must be provided.
`